tests/setup.js

```diff
@@ -7,8 +7,12 @@ configure({ adapter: new Adapter() });
 
 global.confirm = jest.fn();
 
-jest.spyOn(global.console, 'log').mockImplementation(() => jest.fn());
-jest.spyOn(global.console, 'warn').mockImplementation(() => jest.fn());
+if (!process.env.hasOwnProperty('FIDDLE_VERBOSE_TESTS')) {
```

Comment:
👍 on this, but could we document it under `CONTRIBUTING.md`?

Reply:
Great idea. Added 63ecea476bcb72e323c44c471acfcc0c10edd97e